### DISCUSSION TITLE
HSTSを無効にすることはできますか？

### DISCUSSION THREAD

**Discussion by MinaiBytes** [growilabs/growi#7514] [Help / Questions]
アクセス時のレスポンスヘッダーに
```
Strict-Transport-Security: max-age=15552000; includeSubDomains
```
が含まれています。
これを無効化 ( or max-age=0)にすることはできますか？

**Comment by MinaiBytes** [comment]
補足ですが、今は無理やりnginxを挟んで`proxy_hide_header`して消しています。

**Comment by yuki-takei** [comment]
[helmet](https://helmetjs.github.io/) のデフォルト挙動のようですね。

GROWI ではこのように利用しているようで、オプションは変更可能にはなっていないです。
https://github.com/weseek/growi/blob/4f70a9f1e53c243c74e6253cd8adfc402057925b/packages/app/src/server/crowi/express-init.js#L63-L68

環境変数経由で helmet のオプションを上書きするような PR を出していただければマージを検討したいですが、いかがでしょうか。
